docs/2.9.0/docs/canton/usermanual/upgrading.rst

```diff
@@ -223,6 +251,36 @@ command.
 Note that the ``migrate_domain`` command in 2.9 now automatically removes such data,
 but only for the sync domain on which it has been invoked.
 
+Paging in party Management
+""""""""""""""""""""""""""
+By default, the `ListKnownParties` method on the `PartyManagementService` will now only return at most 10'000
+to avoid memory issues in participants that knows more than 10'000 parties.
+
+The `next_page_token` can be used to request the next page (see `request <https://docs.daml.com/2.9.0/app-dev/grpc/proto-docs.html#com-daml-ledger-api-v1-admin-listknownpartiesrequest>`_
+and `response <https://docs.daml.com/2.9.0/app-dev/grpc/proto-docs.html#com-daml-ledger-api-v1-admin-listknownpartiesresponse>`_).
+
+Executor Service Metrics removed
+""""""""""""""""""""""""""""""""
+The metrics for the execution services have been removed:
```

Comment:
@mziolekda @atriantafyllos-da are there alternative metrics a customer should now monitor given these got removed? or the customer is supposed to remove those metrics from their monitoring? in the latter case, isn't there a blind spot?

Reply:
The metrics were behind a quite prohibitive flag since 2.7 and got removed in 2.8. So this issue should not be mentioned on the 2.9 upgrading manual.

Reply:
OK, thanks, will remove.